### PR TITLE
feat(BrandMono): Port component into Brand package

### PR DESCRIPTION
**Pull request by rogervera**
## Port  over<RobotoMono/> wrapper component from Monolith

Currently, Codecademy uses `Monaco` as our default monospaced font. However, there are a few Brand related instances (such as Certificates, Learner Stories, Editorial pages) where we are using
`Roboto Mono`. 

**See this internal Slack thread for history on discussion:** https://codecademy.slack.com/archives/C047ESP1T/p1583765381031600

Reasons:
1. For work on moving Editorial Template components into Brand Packages ([DISCO-125](https://codecademy.atlassian.net/browse/DISCO-125)), Roboto Mono will need to be in client-modules.
2. If we eventually adopt RobotoMono, this will be a change in client-modules anyway. So bringing this one step closer.

### Monolith Use example
Screen Shot 2020-04-29 at 9 24 35 AM

### Storybook example
Screen Shot 2020-04-29 at 9 37 54 AM

---

## Merging your changes

When you are ready to merge to master and publish your changes, use the "squash and merge" button in GitHub, and follow the [PR Title Guide](https://github.com/Codecademy/client-modules#pr-title-guide) to format your PR title and write your PR merge description.

**IMPORTANT:** If your PR contains breaking changes, please remember to follow the instructions for breaking changes!

### CODE REVIEW

File: packages/styleguide/stories/Brand/RobotoMono/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import "~@codecademy/gamut-styles/utils";
+
+.fontName {
+  color: $brand-purple;
+}
+.monacoFont {
+  font-family: $font-monospace;
```
Comment (judahtanthony) [inline packages/styleguide/stories/Brand/RobotoMono/styles.module.scss:7]: Should we have a font variable for roboto as well?

File: packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      <Heading fontSize="xl" as="h1">
+        Roboto Mono
+      </Heading>
+      <Heading fontSize="lg" as="h2">
+        {text('text', 'The quick brown fox jumps over the lazy dog')}
+      </Heading>
+    </RobotoMono>
+    <Heading fontSize="xl" as="h1" className={styles.monacoFont}>
+      Monaco
+    </Heading>
```
Comment (judahtanthony) [inline packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx:44]: Are you leaving Monaco in here for comparison?  Or is this left over from the port?

File: packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+} from '../../Templating';
+
+export default decoratedStories('Brand', RobotoMono);
+
+export const quote = decoratedStory(() => (
+  <StoryTemplate status={StoryStatus.Ready}>
+    <StoryDescription>
+      A wrapper component that will style text with the
+      <RobotoMono>
+        <span> Robot Mono </span>
```
Comment (judahtanthony) [inline packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx:23]: Why the span?  Doesn't the component provide the span?

File: packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx (hunk 2 above)
Comment (rogervera) [reply packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx:44]: Leaving it here for comparison!

File: packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx (hunk 3 above)
Comment (rogervera) [reply packages/styleguide/stories/Brand/RobotoMono/RobotoMono.stories.tsx:23]: True, left over from some styling I had played with. Will remove.

File: packages/styleguide/stories/Brand/RobotoMono/styles.module.scss (hunk 1 above)
Comment (rogervera) [reply packages/styleguide/stories/Brand/RobotoMono/styles.module.scss:7]: Talked about holding that off for now here. https://codecademy.slack.com/archives/C047ESP1T/p1583765381031600

A search for `<RobotoMono/>` in the monolith app shows a few places it's being used as per the story behavior right now. This PR is just for porting that component, not adding the font variable.

**Comment by rogervera** [comment]
> Did you mean to export this as a named story called "quote"? Is this a copy-paste artifact from packages/styleguide/stories/Brand/Quote.stories.tsx?

Thanks for the quick review Judah! Implemented changes in the latest commit.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.12-alpha.5a4b7a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea99fb7af00a4610b50e2de--codecademy-styleguide.netlify.app>https://5ea99fb7af00a4610b50e2de--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea99fb7af00a4610b50e2de>Deploy Logs</a></p>

**Comment by rogervera** [comment]
> Nice 👍

Per offline discussion with @jcenglish and @jakemhiller , renaming this to a more general `BrandMono`

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.1.1-alpha.2f2208.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea9c500a56099656afd245b--codecademy-styleguide.netlify.app>https://5ea9c500a56099656afd245b--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea9c500a56099656afd245b>Deploy Logs</a></p>

**Pull request merged**